pyqtgraph/graphicsItems/DateAxisItem.py

```diff
@@ -213,46 +232,82 @@ def tickValues(self, minVal, maxVal, minSpc):
 ], "99:99:99")
 
 
-def getOffsetFromUtc():
+def fromSecsSinceEpoch(timestamp):
+    try:
+        return QDateTime.fromSecsSinceEpoch(round(timestamp))
+    except OverflowError:
+        return QDateTime()
+
+
+def calculateUtcOffset(timestamp):
+    return -fromSecsSinceEpoch(timestamp).offsetFromUtc()
```

Comment:
negative, because previous implementation returned `time.timezone` or `time.altzone`, which are negative if the local DST timezone is **east** of UTC, and Qt returns negatives for west